.github/workflows/build-images.yml

```diff
@@ -0,0 +1,86 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+  release:
+  workflow_dispatch:
```

Comment:
Is that used?

Reply:
You mean the workflow_dispatch? 
I like the option to start the pipeline per hand.
Do you think it's not needed?

What do you prefer the on release or on tag? (The tag also happen on a release I think)

Reply:
I prefer to use `release`, as this is the way new official versions are produced, and yes, I also believe a release will also trigger a tag event.
Yes, I meant the `workflow_dispatch`. Yes, starting the pipeline by hand is always handy (haha), but I am not sure whether `workflow_dispatch` is needed for that.

Reply:
Yeah its needed to define there
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch